dali/dfu/dfurun.cpp

```diff
@@ -595,12 +595,57 @@ class CDFUengine: public CInterface, implements IDFUengine
             auditflags |= DALI_LDAP_WRITE_WANTED;
         SecAccessFlags perm = queryDistributedFileDirectory().getFDescPermissions(fd,user,auditflags);
         IDFS_Exception *e = NULL;
-        if (!HASREADPERMISSION(perm))
+        if (!write&&!HASREADPERMISSION(perm))
             throw MakeStringException(DFSERR_LookupAccessDenied,"Lookup permission denied for physical file(s)");
         if (write&&!HASWRITEPERMISSION(perm))
             throw MakeStringException(DFSERR_CreateAccessDenied,"Create permission denied for physical file(s)");
     }
 
+    void checkPlaneFilePermissions(IFileDescriptor *fd,IUserDescriptor *user,bool write)
+    {
+        //This function checks the scope permission for one or multiple plane files.
+        //For the multiple files case, all files must be in the same directory (scope).
+        //The IFileDescriptor is used to retrieve the directory path, as well as
+        //the plane name. It also be used for the legacy check on a physical file format
```

Comment:
Can we reword as :  

This function checks the scope permissions for a file or files that reside in a single directory on a single plane.
The IFileDescriptor is used to discover the plane and directory.
If the plane is not present, it implies that it is a bare-metal system and useDropZoneRestriction is off, and there is no matching dropzone in the environment.
If this is the case, or the plane permissions are not found and @failOverToLegacyPhysicalPerms is configured, then the legacy physical file permissions will be checked.